src/printer_schema.h

```diff
@@ -20,6 +20,25 @@
 
 #include "tree_schema.h"
 
+
+/**
+ * @defgroup schemaprinterflags Schema output options
+ * @ingroup schema
+ *
+ * @{
+ */
+#define LYS_OUTPUT_NO_SUBST          0x10 /**< Print only top-level/referede node information,
```

Comment:
I suggest  using  `LYS_OUTPUT_NO_SUBSTMT` in order to be consistent with other place.

Reply:
agreed